views/includes/save-and-continue-button.html

```diff
@@ -1,7 +1,13 @@
 {% set textContinueButton = "Continue" %}
 
-{% if OE_CONFIGS.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 == "true" or OE_CONFIGS.FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME == "true" %}
-  {% set textContinueButton = "Save and continue" %}
+{% if ('/update-an-overseas-entity' in backLinkUrl) %}
```

Comment:
Using the URL here is a little bit 'brittle'. I'd say it's acceptable as long as a test - or tests - start failing if the URL is changed [here](https://github.com/companieshouse/overseas-entities-web/blob/67b0c3ab3e5a1bc0f50cf7b732814ed9ba837fee/src/config/index.ts#L164) but not in this view (alerting developers that a change is needed here too).

Reply:
Thats a good point! `OE_CONFIGS.UPDATE_LANDING_URL` would be the better approach.